packages/client/src/components/formConfig/formTools/CustomFieldTools.tsx

```diff
@@ -769,6 +770,8 @@ class CustomFieldToolsComp extends React.Component<
                     return
                   }
                   const modifiedField = this.prepareModifiedFormField()
+                  console.log('modifiedField', modifiedField)
+                  console.log('selectedField', selectedField)
```

Comment:
@sadmananik Could you please remove these logs?

Reply:
Sure thing